### PR TITLE
New version: JohnMacFarlane.Pandoc version 3.1.13

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/148127)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-04-08T01:25:46.2639691Z ##[error]Manifest Validation Failed
 2024-04-08T01:25:51.3424898Z ##[section]Finishing: Validate Manifest

(Automated response - build 876.)

File: manifests/j/JohnMacFarlane/Pandoc/3.1.13/JohnMacFarlane.Pandoc.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- Architecture: x64
+  InstallerUrl: https://github.com/jgm/pandoc/releases/download/3.1.13/pandoc-3.1.13-windows-x86_64.msi
+  InstallerSha256: 51DA43AAEB93926921BEB777D18B2362D9B6A785D1333177291BB5E8834A4874
+  InstallerSwitches:
+    Custom: ALLUSERS="2"
+- Architecture: x64
+  InstallerUrl: https://github.com/jgm/pandoc/releases/download/3.1.13/pandoc-3.1.13-windows-x86_64.msi
+  InstallerSha256: 51DA43AAEB93926921BEB777D18B2362D9B6A785D1333177291BB5E8834A4874
+  InstallerSwitches:
+    Custom: ALLUSERS=1
```
Comment (stephengillie) [inline manifests/j/JohnMacFarlane/Pandoc/3.1.13/JohnMacFarlane.Pandoc.installer.yaml:29]: Should one of these have the `user` scope from above, and the other have `machine` scope?

Comment (Exorcism0666) [reply manifests/j/JohnMacFarlane/Pandoc/3.1.13/JohnMacFarlane.Pandoc.installer.yaml:29]: @russellbanks, the scope is missing 😭 .

**Comment by stephengillie** [comment]
Hi @Exorcism0666,

> This PR's version number 3.1.13 has 3 parameters (sets of numbers between dots - major, minor, etc), which is less than the current manifest's version 3.1.12.3, which has 4 parameters.

Does this PR's version number **exactly** match the version reported in the Apps & features Settings page? (Feel free to attach a screenshot.)

(Automated response - build 876)

**Comment by Exorcism0666** [comment]
> Hi @Exorcism0666,
> 
> > This PR's version number 3.1.13 has 3 parameters (sets of numbers between dots - major, minor, etc), which is less than the current manifest's version 3.1.12.3, which has 4 parameters.
> 
> Does this PR's version number **exactly** match the version reported in the Apps & features Settings page? (Feel free to attach a screenshot.)
> 
> (Automated response - build 876)

image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**